pyproject.toml

```diff
@@ -7,8 +7,8 @@ packages = [{ include = "fennel" }]
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.8"
-pandas = "^1.0.0"
+python = "^3.9"
```

Comment:
should we remove the CI which runs for 3.8 as well? Given we have declared incompatibility for it?

Reply:
I have already removed it